_data/others.json

```diff
@@ -30,5 +30,14 @@
     "last_payment": 0,
     "all_time": 425,
     "since": "Apr 14, 2019"
+  },
+  {
+    "name": "Works on Arm",
+    "url": "https://developer.arm.com/solutions/infrastructure/works-on-arm",
+    "logo": "sponsors/woa.png",
+    "last_payment": 500,
+    "all_time": 0,
```

Comment:
For posterity, we're not listing the total, so there's no point in having this updated